rest/route/project_copy_test.go

```diff
@@ -0,0 +1,103 @@
+package route
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	restmodel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/stretchr/testify/suite"
+)
+
+type ProjectCopySuite struct {
+	data  data.MockProjectConnector
+	sc    *data.MockConnector
+	route *projectCopyHandler
+
+	suite.Suite
+}
+
+func TestProjectCopySuite(t *testing.T) {
+	suite.Run(t, new(ProjectCopySuite))
```

Comment:
note: for future reference, it might make sense to curtail use of the suite module and try to write more subtests.

Reply:
I wasn't aware we were trying to curtail the suite method. We should talk about this in person at some point.